test/Graphics/Transformable.cpp

```diff
@@ -0,0 +1,123 @@
+#include <SFML/Graphics/Transformable.hpp>
+#include "GraphicsUtil.hpp"
+
+#include <doctest.h>
+
+using doctest::Approx;
+
+TEST_CASE("sf::Transformable class - [graphics]")
+{
+    SUBCASE("Construction")
+    {
+        const sf::Transformable transformable;
+        CHECK(transformable.getPosition() == sf::Vector2f(0, 0));
+        CHECK(transformable.getRotation() == 0);
+        CHECK(transformable.getScale() == sf::Vector2f(1, 1));
+        CHECK(transformable.getOrigin() == sf::Vector2f(0, 0));
+        CHECK(transformable.getTransform() == sf::Transform());
+        CHECK(transformable.getInverseTransform() == sf::Transform());
+    }
+
+    SUBCASE("Setters and getters")
+    {
+        sf::Transformable transformable;
+
+        CHECK(transformable.getPosition() == sf::Vector2f(0, 0));
```

Comment:
We tested this already above. Same for rotation and scale.

Reply:
I was on the fence about those precondition asserts but I'll remove them.

Reply:
I would avoid testing the same thing twice. It makes different test cases dependent on each other.